tests/core/test_transport.py

```diff
@@ -3,16 +3,27 @@
 """Test QMI_Transport functionality."""
 import os, sys
 import socket
+import asyncio
 import threading
 import time
 import unittest.mock
-import vxi11
 import warnings
+import random
+import string
+
+import vxi11
 
 import qmi
 import qmi.core.exceptions
 import qmi.core.transport
-from qmi.core.transport import QMI_TcpTransport, QMI_SerialTransport, QMI_Vxi11Transport, QMI_UsbTmcTransport
+from qmi.core.transport import (
+    QMI_UdpTcpTransportBase,
```

Comment:
can remove this import

Reply:
I need it now after adding tests for comments above.